### DISCUSSION TITLE
Option to enable or disable the new feed

### DISCUSSION THREAD

**Discussion by WillTheDeveloper** [community/community#13153] [Other Feature Feedback, Questions, & Ideas] [answered]
The new feed design is really cool but I also like the single feed, having an option to enable and disable the TWO would be awesome!

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
